ui/src/ScholarReader.tsx

```diff
@@ -430,14 +417,18 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
     const SCROLL_OFFSET_X = -400;
     const SCROLL_OFFSET_Y = +100;
 
-    if (this.state.pdfViewer !== null) {
+    if (this.state.pdfViewer !== null && this.state.pages !== null && this.state.pages[box.page + 1] !== undefined) {
```

Comment:
I think the checks `this.state.pages !== null && this.state.pages[box.page + 1] !== undefined` might be overly restrictive. Pages only get added to `this.state` when pdf.js tells us that the page has been rendered, which will only happen when the page scrolls into focus.

I think it will be frequent that `jumpToBoundingBox` will be called for symbol on pages that hasn't yet been rendered by pdf.js. So if we can, this condition should be removed.

Reply:
Yes agreed but unfortunately ts yells at me without this here :(